tests/test_fetch_same.py

```diff
@@ -0,0 +1,65 @@
+from nose.tools import assert_true, assert_false, assert_equal, \
+                        assert_list_equal, raises
+from . import PREFIX, CONN_INFO
+import datajoint as dj
+# import importlib
+# dj = importlib.import_module('datajoint-python.datajoint', None)
+import numpy as np
+
+Schema = dj.schema(PREFIX + '_fetch_same', connection=dj.conn(**CONN_INFO))
+
+
+@Schema
+class ProjData(dj.Manual):
+    definition = """
+    id : int
+    ---
+    resp : float
+    sim  : float
+    big : longblob
+    blah : varchar(10)
+    """
+
+data_ins = [
```

Comment:
use complete English words for variable names.

Reply:
OK